python/pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[project]
+name = "ccao"
+version = "1.3.0"
```

Comment:
Keeping the version the same as the R package, since they'll have to share the same git tags anyway.